iceoryx_posh/test/moduletests/test_mepoo_mempool.cpp

```diff
@@ -113,10 +113,19 @@ TEST_F(MemPool_test, freeChunk)
         chunks.push_back(reinterpret_cast<uint8_t*>(sut.getChunk()));
     }
 
-    for (uint32_t i = 0; i < NumberOfChunks; ++i)
+    for (uint32_t i = 0; i <= NumberOfChunks; ++i)
     {
-        EXPECT_THAT(sut.getUsedChunks(), Eq(NumberOfChunks - i));
-        sut.freeChunk(chunks[i]);
+        if(i==NumberOfChunks)
```

Comment:
This line looks like `clang-format` is not working properly in your setup. Can you please check?!